### ISSUE TITLE
Invalid flag for install location local

### ISSUE THREAD

**Issue by wyqydsyq** [nearbydelta/itypescript#2]
Following the readme, when I try to install locally running:
```
its --ts-install=local
```
I get the error
```
Invalid flag for install location local
```

**Comment by nearbydelta** [comment]
Sorry for the late reply. The bug will be fixed in the next version.
For manual fix, you can change the code as follows:

The line 106 & 107 in `NPM/LIBRARY/PATH/itypescript/bin/itypescript.js`:
**BEFORE:**
```javascript
    InstallLoc[InstallLoc["local"] = 0] = "local";
    InstallLoc[InstallLoc["global"] = 1] = "global";
```
Replace them with the following lines:
**AFTER**:
```javascript
    InstallLoc[InstallLoc["local"] = 1] = "local";
    InstallLoc[InstallLoc["global"] = 2] = "global";
```

It was the bug because of wrong indexing of enumeration.
Again, sorry for the late reply.

**Comment by defel** [comment]
nice, this worked! Is it possible to publish a release-candidate or something with this bugfix until next version arrives? :)

**Comment by nearbydelta** [comment]
The issue resolved in version 0.2.2.
